src/styles/calendar.scss

```
@@ -0,0 +1,52 @@
+.calendar {
+  width: $col-count * $cell-size + ($col-count - 1);
+  padding: 20px;
+  border: 1px solid #000;
+  border-radius: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+}
+
+.calendar__day {
```

```suggestion
.calendar {
  width: $col-count * $cell-size + ($col-count - 1);
  padding: 20px;
  border: 1px solid #000;
  border-radius: 10px;
  display: flex;
  flex-wrap: wrap;
  gap: 1px;

  &__day {
```

Make use of SASS nesting (checklist p4)